test/integration/listener_lds_integration_test.cc

```diff
@@ -64,43 +67,13 @@ class ListenerIntegrationTest : public HttpIntegrationTest,
       ConfigHelper::setHttp2(*rds_cluster);
     });
 
-    config_helper_.addConfigModifier(
-        [this](
-            envoy::extensions::filters::network::http_connection_manager::v3::HttpConnectionManager&
-                http_connection_manager) {
-          auto* rds_config = http_connection_manager.mutable_rds();
-          rds_config->set_route_config_name(route_table_name_);
-          rds_config->mutable_config_source()->set_resource_api_version(
-              envoy::config::core::v3::ApiVersion::V3);
-          envoy::config::core::v3::ApiConfigSource* rds_api_config_source =
-              rds_config->mutable_config_source()->mutable_api_config_source();
-          rds_api_config_source->set_api_type(envoy::config::core::v3::ApiConfigSource::GRPC);
-          rds_api_config_source->set_transport_api_version(envoy::config::core::v3::V3);
-          envoy::config::core::v3::GrpcService* grpc_service =
-              rds_api_config_source->add_grpc_services();
-          setGrpcService(*grpc_service, "rds_cluster", getRdsFakeUpstream().localAddress());
-        });
-
+    setUpGrpcRds();
     // Note this has to be the last modifier as it nuke static_resource listeners.
     setUpGrpcLds();
+    ENVOY_LOG_MISC(debug, "listener config: {}", listener_config_.DebugString());
```

Comment:
Don't we already print this somewhere? I thought we do so just want to make sure we aren't printing this twice?

Reply:
yea, I add it when I debug, then I forget to remove it, let me cleanup it.